subjects/java/checkpoints/time-tracker/README.md

```diff
@@ -0,0 +1,72 @@
+## TimeTracker
+
+### Instructions
+
+Write a `ProjectTime` class to track the duration of a project using start and end times. You will be asked to add getter and setter methods for the `startTime` and `endTime` attributes. When you set one of them, you must update the `hoursLogged` attribute. The `hoursLogged` getter should output the hours in the following format:
+
+- Less than 120 minutes: `hoursLogged` minutes
+- Less than 120 hours: `hoursLogged` hours
+- Less than 120 days: `hoursLogged` days
+- More than 120 days: `hoursLogged` months
```

Comment:
In the provided examples, `minutes` is `m`, `hours` is `h` ... I would keep this consistent since there are no examples for `days` and `months`: how are these two supposed to be identified?

Reply:
Yeah, you are right i will fix that.